content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md

```diff
@@ -40,3 +41,46 @@ This results in the following output:
 11
 0
 ```
+
+## Codebyte Examples
+
+### Example with a specific date and formatting
+
```

Comment:
Add a small introductory sentence here above this codebyte which can give rough idea to the reader of what to expect from the code byte. Do the same for the cobytes below this too.

Reply:
working on it...

Reply:
updated